CONTRIBUTING.rst

```diff
@@ -40,3 +44,59 @@ To bring in updated upstream modules.  Update `gilt.yml` and execute the followi
 
 .. _`Ansible Goss`: https://github.com/indusbox/goss-ansible
 .. _`Gilt`: http://gilt.readthedocs.io
+
+
+Move to Red Hat
```

Comment:
I do not see a reson to transform documentation in a piece of written history. If someone wants history, they can check the git history. Current documentatin should be about present state of the project.